packages/rs-sdk/src/platform/transition/waitable.rs

```diff
@@ -0,0 +1,119 @@
+use std::collections::BTreeMap;
+
+use super::broadcast::BroadcastStateTransition;
+use super::put_settings::PutSettings;
+use crate::platform::Fetch;
+use crate::Error;
+use crate::Sdk;
+use dpp::document::Document;
+use dpp::prelude::{DataContract, Identifier, Identity};
+use dpp::state_transition::identity_create_transition::accessors::IdentityCreateTransitionAccessorsV0;
+use dpp::state_transition::StateTransition;
+use dpp::state_transition::StateTransitionLike;
+use dpp::ProtocolError;
+
+/// Waitable trait provides a wait to wait for a response of a state transition after it has been broadcast and
+/// receive altered objects.
+///
+/// This is simple conveniance trait wrapping the [`BroadcastStateTransition::wait_for_response`] method.
+#[async_trait::async_trait]
+pub trait Waitable: Sized {
```

Comment:
I'm not sure this structure is the best, when we have all our entities used in one module. I think we should split code by domain entities: document related modules where we have everything about documents and so on. Let's discuss and create a task